src/frontend/src/components/Menu.tsx

```diff
@@ -96,13 +106,41 @@ const Menu = ({ languages, compareCount }: Props) => {
                         </Button>
                     </>
                 ) : (
-                    // TODO: Implement the hamburger menu, pending design
-                    <Button
-                        variant="ghost"
-                        leftIcon={
-                            <HamburgerIcon className="fill fill-teal-900 text-sm h-5 w-5 font-normal" />
-                        }
-                    />
+                    <>
+                        <Button
+                            variant="ghost"
+                            leftIcon={
+                                <HamburgerIcon className="fill fill-teal-900 text-sm h-5 w-5 font-normal" />
+                            }
+                            onPress={() => setIsOpen(true)}
```

Comment:
Came across this: https://react-spectrum.adobe.com/blog/building-a-button-part-1.html and thus started to use `onPress` instead of `onClick` under the React Aria context